server/models/serviceProviderSentReferralSummary.ts

```diff
@@ -7,4 +7,5 @@ export default interface ServiceProviderSentReferralSummary {
   serviceUserFirstName: string | null
   serviceUserLastName: string | null
   endOfServiceReportSubmitted?: boolean
+  concluded: boolean
```

Comment:
Do you think we should add this to the `sentReferralSummary` pact tests too? We can probably do that by using the updated factory here: https://github.com/ministryofjustice/hmpps-interventions-ui/blob/21aa181241b9c9b24b3f49d43ac8747293e400de/server/services/interventionsService.test.ts#L1693 instead of creating one from scratch. Would that break the Interventions Service pacts?

Reply:
Good point -- I'll have to do this change after the back-end has been approved.